src/MockData/cities.json

```diff
@@ -0,0 +1,962 @@
+[
+  {
+    "id": 0,
+    "name": "Tripoli",
+    "code": "tri7",
+    "description": "Tripoli is the largest city in northern Lebanon and the second-largest city in the country",
+    "lat": 34.46664758010337,
+    "lng": 35.831724876807975,
+    "address_coordinates": [34.46664758010337, 35.831724876807975],
+    "country": "Lebanon",
+    "iso2": "LB",
+    "admin_name": "Liban-Nord",
+    "capital": "admin",
+    "population": "530000",
+    "population_proper": "192572",
+    "image-array": [
+      "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQcWgUCkF5WKebdRgPH7cB7S-cGyarWbh_9NwTwGyGCsNJI6gXq1lxkTS9-h9aQRrcXlBI&usqp=CAU",
+      "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcT4RJiZWVAsA4YkYFlxCyft5uI_pfDnW-41zg&usqp=CAU",
+      "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRfnSoLCMsJYalhtzYl50TXCUkZNBRAmrJTdw&usqp=CAU",
+      "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTDaQi2TQSsCFQRMoWKgSxCCZW3jHHPv9vREQ&usqp=CAU",
+      "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcT4uQTJxIv1DoFAeHZj9Kxo4pIxAhP_W0Ng1g&usqp=CAU"
+    ],
+    "image": "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSeUB6kWjsLzQzBDpUgBswD8jvLlffLX8S_pQ&usqp=CAU",
+    "activity": [
```

Comment:
this shouldn't be here but rather when you go to the city page you will fetch the activities for this city, each activity should have a city so we can filter according to that